templates/plugin/kafkainput.toml.erb

```diff
@@ -0,0 +1,42 @@
+# This file is controlled via puppet.
+[<%= @plugin_name -%>]
+type = "KafkaInput"
+<%= scope.function_template(['heka/plugin/_input.toml.erb']) %>
```

Comment:
In order to be consistent with the output templates, this should be moved to the end of the file.